python/variable.cpp

```diff
@@ -69,7 +69,7 @@ void bind_init_0D_native_python_types(py::class_<Variable> &c) {
                                                         dtype);
           }
         }),
-        py::arg("value"), py::arg("variance") = std::nullopt,
+        py::arg("value").noconvert(), py::arg("variance") = std::nullopt,
```

Comment:
Is there also a problem with non-scalar Variables?

Do we always have a perfect match for all possible dtypes? Thinking of `str` or `np.datetime64`.

Reply:
This function is only used for `bool`, `int64_t` and `bool`.